sdk/ml/azure-ai-ml/azure/ai/ml/operations/_component_operations.py

```diff
@@ -495,7 +496,7 @@ def preprocess_job(node):
                     # Put binding string to field
                     setattr(node, field_name, val._data_binding())
```

Comment:
Add to supported fields above if we are going to support binding on hdi_node.compute_name

Reply:
... Thank you for pointing out this. Maybe we can think about if there is a better way to organize such logic later